iamproperty/iamproperty.github.io
# Colour page added

## Summary of Changes:
1. Colours doc page added
2. Renamed some sass folders

## Rationale
Create a page to view all of the colours within the framework and how they can be used.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /foundations/colours

## Ticket(s)
DS-22

## Checklist

## Notes
Some of these colours are subject to change
